ansible/roles/gpu/tasks/nvidia-gpu-Flatcar.yaml

```diff
@@ -1,18 +1,197 @@
 ---
-- name: Load loop modules
-  modprobe:
-    name:
-      - loop
-      - i2c_core
-      - ipmi_msghandler
-    state: present
-
-# TODO: Replace me with proper ansible resources 
-- name: Load loop module
+- name: create /opt/lib directory
+  file:
+    path: "/opt/lib/"
+    state: directory
+
+- name: create /opt/var/log directory
+  file:
+    path: "/opt/var/log/"
+    state: directory
+
+# build kernel modules
+- name: load loop modules
+  command: modprobe -d / -a loop i2c_core ipmi_msghandler
```

Comment:
Can we not use the `modprobe` module here?

Reply:
That did not work for me for reasons that are beyond me.

Reply:
Should try again...

Reply:
...actually no. I will leave a comment on top of this one indicating that we might wanna try again or find out why this one does not work. IIUC then specifying `/` means it will look at standard locations, thus ansible modprobe should be able to get it done as well. But it did not in my and Jan's testing.